src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    private List<Truck> trucks = new ArrayList<>();
+
+    public TruckProducer() {
+
+    }

```

Comment:
We do not need explicitly create no-args constructor. It is available as default in case no other constructors are present
```suggestion
```